subxt/src/backend/mod.rs

```diff
@@ -336,54 +336,96 @@ pub struct StorageResponse {
 #[cfg(test)]
 mod test {
     use super::*;
+    pub use crate::{backend::StorageResponse, error::RpcError};
+    pub use futures::StreamExt;
+    pub use serde::Serialize;
+    pub use std::collections::{HashMap, VecDeque};
+    pub use subxt_core::{config::DefaultExtrinsicParams, Config};
+    pub use tokio::sync::{mpsc, Mutex};
+    pub type RpcResult<T> = Result<T, RpcError>;
+    pub type Item = RpcResult<String>;
+    pub use crate::backend::rpc::RawRpcSubscription;
+    pub use rpc::RpcClientT;
+    pub use serde_json::value::RawValue;
+
+    fn storage_response<K: Into<Vec<u8>>, V: Into<Vec<u8>>>(key: K, value: V) -> StorageResponse
+    where
+        Vec<u8>: From<K>,
+    {
+        StorageResponse {
+            key: key.into(),
+            value: value.into(),
+        }
+    }
+    pub mod rpc_client {
 
-    mod legacy {
-        use super::rpc::{RpcClient, RpcClientT};
-        use crate::backend::rpc::RawRpcSubscription;
-        use crate::backend::BackendExt;
-        use crate::{
-            backend::{
-                legacy::rpc_methods::Bytes, legacy::rpc_methods::RuntimeVersion,
-                legacy::LegacyBackend, StorageResponse,
-            },
-            error::RpcError,
-        };
-        use futures::StreamExt;
-        use serde::Serialize;
-        use serde_json::value::RawValue;
-        use std::{
-            collections::{HashMap, VecDeque},
-            sync::Arc,
-        };
-        use subxt_core::{config::DefaultExtrinsicParams, Config};
-        use tokio::sync::{mpsc, Mutex};
+        use std::time::Duration;
+
+        use super::*;
 
```

Comment:
```suggestion

    pub mod rpc_client {
        use std::time::Duration;
        use super::*;

```

Nit: for consistency with above

Reply:
Done